django_project/cplus_api/migrations/0003_userroletype_userprofile.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.7 on 2024-04-09 01:34
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('cplus_api', '0002_outputlayer_inputlayer'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserRoleType',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=50, unique=True)),
+                ('description', models.TextField()),
+            ],
+            options={
+                'verbose_name': 'User role',
+                'verbose_name_plural': 'User roles',
+                'db_table': 'user_role',
+            },
+        ),
+        migrations.CreateModel(
+            name='UserProfile',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('role', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='role_profile', to='cplus_api.userroletype')),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='user_profile', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
```

Comment:
@zamuzakki could we initialize the UserRoleType with two roles (External and Internal) in the migrataion file? thanks

Reply:
Updated  @danangmassandy